src/kv/bulk.rs

```diff
@@ -0,0 +1,67 @@
+extern crate base64;
+
+use cloudflare::endpoints::workerskv::delete_bulk::DeleteBulk;
+use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
+use cloudflare::endpoints::workerskv::write_bulk::WriteBulk;
+use cloudflare::framework::apiclient::ApiClient;
+
+use crate::commands::kv;
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::toml::Target;
+
+const MAX_PAIRS: usize = 10000;
+
+pub fn put(
+    client: &impl ApiClient,
+    target: &Target,
+    namespace_id: &str,
+    pairs: &[KeyValuePair],
+) -> Result<(), failure::Error> {
+    // Validate that bulk upload is within size constraints
+    if pairs.len() > MAX_PAIRS {
+        failure::bail!(
+            "Number of key-value pairs to upload ({}) exceeds max of {}",
+            pairs.len(),
+            MAX_PAIRS
+        );
+    }
+
+    match client.request(&WriteBulk {
+        account_identifier: &target.account_id,
+        namespace_identifier: namespace_id,
+        bulk_key_value_pairs: pairs.to_owned(),
+    }) {
+        Ok(_) => Ok(()),
+        Err(e) => failure::bail!("{}", kv::format_error(e)),
```

Comment:
in another section, we imported this and just called `format_error` - lets make em consistent

Reply:
ugh i  hate that this is hereeeeeeeee next stop `thiserror`